wagtail_localize/compat.py

```diff
@@ -14,55 +13,28 @@
 
 
 def get_snippet_list_url(snippet):
-    if WAGTAIL_VERSION >= (4, 0):
-        return reverse(
-            f"wagtailsnippets_{snippet._meta.app_label}_{snippet._meta.model_name}:list"
-        )
-
-    reverse(
-        "wagtailsnippets:list", args=[snippet._meta.app_label, snippet._meta.model_name]
+    return reverse(
+        f"wagtailsnippets_{snippet._meta.app_label}_{snippet._meta.model_name}:list"
```

Comment:
now that we no longer need the conditionals, we can drop `compat.py` and replace all the usages with the plain `reverse`

Reply:
I didn't delete `compat.py` completely, as the `DATE_FORMAT` is used by multiple files (e.g. `models.py` and `edit_localize.py`), but the rest of the code, I removed.